src/xtgeo/well/well1.py

```diff
@@ -29,9 +33,20 @@
 # METHODS as wrappers to class init + import
 
 
+def _data_reader_factory(file_format):
+    if file_format in ["rmswell", "irap_ascii"]:
```

Comment:
`rms_well`, the old default is not supported anymore?

Reply:
The data reader factory  `_data_reader_factory` only accept generic file formats (return value from `generic_format_by_proposal` so I guess some documentation should be added here, and maybe `rms_well` added to `SUPPORTED_FORMATS["rmswell"]``.

Reply:
Actually, I cannot find any use of `rms_well` anywhere.